src/domain/resource/resourceKeyboardReservation/ResourceKeyboardSlotPicker.js

```diff
@@ -291,8 +291,8 @@ ResourcePageKeyboardTimePicker.propTypes = {
     end: PropTypes.string,
   }),
   t: PropTypes.func.isRequired,
-  maxPeriod: PropTypes.string.isRequired,
-  minPeriod: PropTypes.string.isRequired,
+  maxPeriod: PropTypes.string,
```

Comment:
Off-topic: these prop types should not be marked `isRequired`, causing console errors.